.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,6 +1,7 @@
 ---
 name: Bug report
 about: Create a report to help us improve
+labels: bug
```

Comment:
Maybe we should have the `bug` label added by whoever is triaging issues instead of adding it automatically? I'm worried that this will lead to issues being falsely tagged as bugs (e.g. if they're really feature requests). Some ideas:

1. It could be the triager's job to check the labels and change them as needed
2. We could add a `triage needed` label to mark the issue for triage
3. We could have a `tentative bug` label that indicates that triage is still needed
4. We could add both the `bug` label and the `triage needed` label so that the triager checks that the `bug` label was correctly added

More broadly, I think this labeling should be part of a broader issue management plan for all of oppia-web

Reply:
Thanks @U8NWXD, I agree! I've gone ahead with (4) for now. Instead of "triage needed" we will be using the "TODO: triage" which is an existing label.